### PR TITLE
Clean up BuildFiles under CondCore/

### PR DESCRIPTION
**Pull request by fwyzard**
Clean up BuildFile.xml filea with `xmllint --format`.

### CODE REVIEW

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by jfernan2** [comment]
+1
Changes due to https://github.com/cms-sw/cmssw/pull/29628

File: CondCore/Utilities/plugins/BuildFile.xml [hunk 1]
```diff
@@ -1,22 +1,25 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (silviodonato) [inline CondCore/Utilities/plugins/BuildFile.xml:1]: @fwyzard

**Comment by silviodonato** [comment]
merge

**Comment by silviodonato** [comment]
merge

**Pull request merged**